tf/tfprocess.py

```diff
@@ -191,9 +191,9 @@ def init_v2(self, train_dataset, test_dataset, validation_dataset=None):
         if self.strategy is not None and validation_dataset is not None:
             self.validation_dataset = self.strategy.experimental_distribute_dataset(validation_dataset)
         else:
-            self.validation_dataset = validation_dataset
-        this = self
+            self.validation_dataset = validation_dataset       
         if self.strategy is not None:
+        this = self
```

Comment:
you forgot the indent - for the python!

Can you run the entire file through yapf? After you fix that?

Reply:
yike stupid mistake. I ran it through yapf, not sure if you wanted a particular style, I left it at default